lib/middleware/request-handler.ts

```diff
@@ -406,11 +412,17 @@ export function elevateRequest (req: Interface.IOurRequest,
         res.write(text + JSON.stringify(data));
     };
 
-    res.sendEnd = (status: string, message: string): void => {
-        var text = util.format('%s%s,%s}',
+    res.sendEnd = (status: string,
```

Comment:
I agree with Eric's point overall. One thing to note, though: using `sendChunk` will result in slightly different output, specifically `correlationIds` will be in the `data` property of the response, rather than as a top level property. I don't think this really matters, though?